.pylintrc

```diff
@@ -157,12 +160,6 @@ disable=abstract-method,
 # mypackage.mymodule.MyReporterClass.
 output-format=text
 
-# Put messages in a separate file for each module / package specified on the
-# command line instead of printing them on stdout. Reports (if any) will be
-# written in a file name "pylint_global.[txt|html]". This option is deprecated
-# and it will be removed in Pylint 2.0.
-files-output=no
```

Comment:
assuming this is removed because its the default

Reply:
Yes, this operation is removed, and it seems google's official style guide for python has removed it as well. : )